CharlieBackend.Data/Repositories/Impl/StudentGroupRepository.cs

```diff
@@ -4,9 +4,10 @@
 using CharlieBackend.Data.Helpers;
 using CharlieBackend.Data.Repositories.Impl.Interfaces;
 using Microsoft.EntityFrameworkCore;
-using System.Collections.Generic;
+using System;
```

Comment:
omg... I remember that some one told me on my first PR that this using unuseful!!! )))

Reply:
that was stupid, agree